Modules/FindNetCDF.cmake

```diff
@@ -170,6 +171,16 @@ function(netcdf_config exec flag output_var)
     endif()
   endif()
 endfunction()
+function(netcdf_config2 exec flag1 flag2 output_var)
+  set(${output_var} False PARENT_SCOPE)
+  if( exec )
+    execute_process( COMMAND ${exec} ${flag1} ${flag2} RESULT_VARIABLE _ret OUTPUT_VARIABLE _val)
+    if( _ret EQUAL 0 )
+      string( STRIP ${_val} _val )
+      set( ${output_var} ${_val} PARENT_SCOPE )
+    endif()
+  endif()
+endfunction()
```

Comment:
Trying this now

Reply:
No, doesn't work:
```
CMake Error at CMakeModules/Modules/FindNetCDF.cmake:186 (netcdf_config):
  netcdf_config Function invoked with incorrect arguments for function named:
  netcdf_config
Call Stack (most recent call first):
  CMakeLists.txt:111 (find_package)
```
So I'll add a one-line documentation above each of the two functions. There is a generic option to have optional arguments, but the syntax looks cryptic and it would require `if` statements inside the functions.